app/src/main/java/com/naman14/timberx/util/Utils.kt

```diff
@@ -33,96 +26,37 @@ import java.util.Collections
 
 object Utils {
 
-    val MUSIC_ONLY_SELECTION = (MediaStore.Audio.AudioColumns.IS_MUSIC + "=1" +
+    const val MUSIC_ONLY_SELECTION = (MediaStore.Audio.AudioColumns.IS_MUSIC + "=1" +
```

Comment:
Now that you mention it, this could be `const val MUSIC_ONLY_SELECTION = """${MediaStore.Audio.BaseColumns.IS_MUSIC = 1 AND ${MediaStore.Audio.AudioColumns.TITLE != ''"""`